testing.rst

```diff
@@ -260,6 +265,17 @@ document::
         $client->getResponse()->getContent()
     );
 
+.. note::
```

Comment:
Let's make this a `tip` instead, shouldn't we?

Reply:
Why not? @javiereguiluz wdyt?

Reply:
ping :)